src/client/bedrock.rs

```diff
@@ -563,6 +585,9 @@ fn aws_fetch(
     let date_stamp = amz_date[0..8].to_string();
     headers.insert("host".into(), host.clone());
     headers.insert("x-amz-date".into(), amz_date.clone());
+    if let Ok(token) = std::env::var("AWS_SESSION_TOKEN") {
+        headers.insert("x-amz-security-token".into(), token);
+    }
```

Comment:
This is the only place that needs to be changed. There is no formatting issue with the text. The `;` is not required.

Reply:
Hardcoding variable names is not good practice. You should add the method `config_get_fn!(session_token, get_session_token);` and use `get_session_token` to get the `session_token`.